pvr.wmc/addon.xml.in

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.wmc"
-  version="2.4.3"
+  version="2.4.5"
```

Comment:
Why is the version number increased by 2?Should read 2.4.4 imo.

Reply:
2.4.4 was used as the version to test. I'm guessing to be safe it was incremented again to ensure any machines with the dev/test version of the add-on get the final version included with Kodi.

Reply:
Hmm, fairly uncommon as now  everybody is wondering what happend to  2.4.4.

Reply:
I'd like to keep the version to 2.4.5.  Basically the for the reason sbthomas said, but there are also comment  changes between the tested version and this one.

Reply:
Also...  I don't know how to 'squash down the changes to a single commit'.  If that's required I would need to do some research, or a little guidance.